.circleci/config.yml

```diff
@@ -134,8 +134,7 @@ jobs:
           name: Install build dependencies
           command: |
             apt-get -qq update
-            apt-get -qy install cmake libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev libboost-program-options-dev libcvc4-dev
-            ./scripts/install_obsolete_jsoncpp_1_7_4.sh
+            apt-get -qy install cmake libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev libboost-program-options-dev libcvc4-dev libjsoncpp-dev=1.7.4-3
```

Comment:
Is there a way to specify the package version ``1.7.4`` without explicitly specifying the build version ``-3`` as well? That'd be nicer - but probably fine either way - I don't expect it to change a lot.

Reply:
I pushed a wildcard version, let's see if that works

Reply:
it worked :)